compute/apiv1/smoke_test.go

```diff
@@ -492,23 +492,24 @@ func TestTypeInt64(t *testing.T) {
 	if err != nil {
 		t.Error(err)
 	}
-
-	defer func(c *ImagesClient, ctx context.Context, req *computepb.DeleteImageRequest) {
-		_, err := c.Delete(ctx, req)
-		if err != nil {
-		}
-	}(c, ctx, &computepb.DeleteImageRequest{
+	delReq := &computepb.DeleteImageRequest{
```

Comment:
Nit: This doesn't need to define this outside of the `defer`, it can be inlined at the function call. My previous mention of that was for regarding the other data, `name` and `projectId`.

Reply:
done